src/pages/Dashboard/Modals/BorrowRepay/Borrow/index.tsx

```diff
@@ -102,6 +116,20 @@ export const BorrowForm: React.FC<IBorrowFormProps> = ({
               hasError={errors.amount === ErrorCode.HIGHER_THAN_MAX}
             />
 
+            <Typography
```

Comment:
nit: I have done this in some places but the `TokenTextField` has a description prop for positioned text at the bottom left of the component.

Reply:
Ah right, I forgot! Good point.